Attendees/Brokers/Storages/StorageBroker.Attendees.cs

```diff
@@ -0,0 +1,34 @@
+using System;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore.Metadata.Builders;
+using Attendees.Models.Foundations.Attendees;
+
+namespace Attendees.Brokers.Storages
+{
+    public partial class StorageBroker
+    {
+        public DbSet<Attendee> Attendees { get; set; }
+
+        public async ValueTask<Attendee> InsertAttendeeAsync(Attendee attendee) =>
+            await InsertAsync(attendee);
+
+        public IQueryable<Attendee> SelectAllAttendees() =>
+            SelectAll<Attendee>();
+
+        public async ValueTask<Attendee> SelectAttendeeByIdAsync(Guid attendeeId) =>
+            await SelectAsync<Attendee>(attendeeId);
+
+        public async ValueTask<Attendee> UpdateAttendeeAsync(Attendee attendee) =>
+            await UpdateAsync(attendee);
+
+        public async ValueTask<Attendee> DeleteAttendeeAsync(Attendee attendee) =>
+            await DeleteAsync(attendee);
+
+        internal void ConfigureAttendees(EntityTypeBuilder<Attendee> builder)
```

Comment:
Method `ConfigureAttendees` violates async broker conventions: return type should be `ValueTask` or `ValueTask<T>`; method name should end with `Async`; Also method should be marked with the `async` modifier.